mempool/Cargo.toml

```diff
@@ -40,6 +38,8 @@ serde_json = "1.0"
 
 libra-proptest-helpers = { path = "../common/proptest-helpers", optional = true }
 proptest = { version = "0.9.6", optional = true }
+
+#[cfg(feature = "fuzzing")]
```

Comment:
You don't need this unless you are just leaving a comment. It is already set up correctly in the `[features]` section below and set as optional here.